servicex/templates/flask_app-configmap.yaml

```diff
@@ -67,6 +67,11 @@ data:
     MINIO_URL_TRANSFORMER = '{{ .Release.Name }}-minio:9000'
     MINIO_ACCESS_KEY = '{{ .Values.minio.accessKey }}'
     MINIO_SECRET_KEY = '{{ .Values.minio.secretKey }}'
+    {{ if .Values.minio.ingress.enabled }}
+    MINIO_PUBLIC_URL = '{{index .Values.minio.ingress.hosts 0 }}'
+    {{ else }}
+    MINIO_PUBLIC_URL = '{{ .Release.Name }}-minio:9000'
```

Comment:
Should this internal name be the name of the service? And shouldn't that be stored in a macro somewhere, rather than generated here? Is the port 9000 settable?

Reply:
I made it possible to override if you don't want this handy default. The 9000 is set inside the Minio sub chart, so now picking it up from that chart's values